app/services/notifications/welcome_notification/send.rb

```diff
@@ -14,15 +14,15 @@ def self.call(*args)
       end
 
       def call
-        dev_account = User.dev_account
+        welcoming_account = User.welcoming_account
         json_data = {
-          user: user_data(dev_account),
+          user: user_data(welcoming_account),
           broadcast: {
             title: welcome_broadcast.title,
             processed_html: welcome_broadcast.processed_html
           }
         }
-        Notification.create(
+        Notification.create!(
```

Comment:
Some additional context for this change here: I ran into a bug locally where it was super unclear why a notification couldn't be created. Once I explicitly raised with `create!`, it was easy to see the issue and fix. Not sure why we _aren't_ raising, but I really feel like we should.